DashKit.xcodeproj/project.pbxproj

```diff
@@ -1390,7 +1388,8 @@
 				CODE_SIGN_STYLE = Manual;
 				COMBINE_HIDPI_IMAGES = YES;
 				DEVELOPMENT_TEAM = "";
-				ENABLE_BITCODE = YES;
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = 75U4X84TEG;
```

Comment:
I thought about using "$(LOOP_DEVELOPMENT_TEAM)", but this project doesn't have access to that xcconfig easily, and this is only being built by Tidepool.

Reply:
Nice